### PR TITLE
Update: `PeterBienstman.Mnemosyne` version. 2.8

### PR DESCRIPTION
**Pull request by vedantmgoyal9**
- [x] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [x] Have you validated your manifest locally with `winget validate --manifest <path>`? 
- [ ] Have you tested your manifest locally with `winget install --manifest <path>`?
- [x] Does your manifest conform to the [1.0 schema](https://github.com/microsoft/winget-cli/blob/master/doc/ManifestSpecv1.0.md)?

Note: `<path>` is the name of the directory containing the manifest you're submitting.

-----

###### Microsoft Reviewers: [Open in CodeFlow](https://portal.fabricbot.ms/api/codeflow?pullrequest=https://github.com/microsoft/winget-pkgs/pull/20821)

### CODE REVIEW

File: manifests/p/PeterBienstman/Mnemosyne/2.8/PeterBienstman.Mnemosyne.installer.yaml [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+PackageIdentifier: PeterBienstman.Mnemosyne
+PackageVersion: 2.8
+MinimumOSVersion: 10.0.0.0
+InstallModes:
+- interactive
+- silent
+- silentWithProgress
+Installers:
+- InstallerLocale: en-US
+  Architecture: x86
```
Comment (palenshus) [inline manifests/p/PeterBienstman/Mnemosyne/2.8/PeterBienstman.Mnemosyne.installer.yaml:13]: Looks like this should actually be x64

**Review by palenshus — CHANGES_REQUESTED** [review]

File: manifests/p/PeterBienstman/Mnemosyne/2.8/PeterBienstman.Mnemosyne.installer.yaml [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+MinimumOSVersion: 10.0.0.0
+InstallModes:
+- interactive
+- silent
+- silentWithProgress
+Installers:
+- InstallerLocale: en-US
+  Architecture: x64
+  InstallerType: inno
+#  Scope:
```
Comment (OfficialEsco) [inline manifests/p/PeterBienstman/Mnemosyne/2.8/PeterBienstman.Mnemosyne.installer.yaml:15]:
```suggestion
  Scope: machine
```

**Review by OfficialEsco — DISMISSED** [review]
Ill let it pass as Publisher and Version is blank
image

**Review by OfficialEsco — APPROVED** [review]
Nice save

**Comment by OfficialEsco** [comment]
Ahh cool, @palenshus broke another PR by requesting a change but not changing the metadata.. (When a maintainer request a change it have to be applied)

**Comment by vedantmgoyal2009** [comment]
All three labels are there but still not merging?

**Comment by vedantmgoyal2009** [comment]
@palenshus Please approve changes so that it can be merged.

**Review by palenshus — APPROVED** [review]

**Pull request merged**